blocks/core/ff_module/ff_module-dropdown-button/ff_module-dropdown-button-component/ff_module-dropdown-button-component.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+var React = require('react');
+var template = require('./_ff_module-dropdown-button-component.rt.js');
+var activateDropdowns = require('../ff_module-dropdown-button');
+
+module.exports = React.createClass({
+    displayName:'DropdownButton',
+    propTypes: {
+        list: React.PropTypes.array.isRequired,
+        id: React.PropTypes.string.isRequired,
+        text: React.PropTypes.string.isRequired
+    },
+    render: template,
+    componentDidMount: function(){
+        activateDropdowns();
```

Comment:
It feels to me as though the least surprising behaviour would be to activate dropdowns just for this component to avoid weird interaction between React and non-React usages of the dropdown. Since `activateDropdowns` lets you both pick the root and the selectors, it seems like that wouldn't be too tricky to implement?